code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+---
+openapi: 3.0.3
+############################################################################
+#                                     API info                             #
+############################################################################
+info:
+  title: Traffic Influence API
+  version: wip
+  description: |
+    ## Overview
+    The reference scenario foresees a Service, composed by one or more Service
+    Producers deployed in different geographical locations on Edge Cloud Zones
+    (Edge datacentres of Telco Operator) or in Cloud. The Service Producer,
+    deployed at the Edge, is referred as Edge Application Server (EAS).\
+    An Edge Cloud Zone is a platform in the Telco Operator network, offering
```

Comment:
"An Edge Cloud Zone is a platform" seems to be proposing a definition here. But we already have defined it at other place and we may use it just An Edge Cloud Zone without further qualifying here in my view.

Reply:
In Edge-Application-Management.yaml I see following definition:
__Edge Cloud Zone__ - An Edge Cloud Zone is the lowest level of
    abstraction exposed to an Application Provider who wants to deploy  an Application on Edge Cloud.
    Edge Cloud Zones exists within a Edge Cloud Region.

So the statement "An Edge Cloud Zone is a platform" is not same as defined in App LCM file. May be we can just use reference of App LCM definition here in TI.